extensions/ql-vscode/src/remote-queries/remote-queries-manager.ts

```diff
@@ -14,16 +14,21 @@ import { RemoteQueryResult } from './remote-query-result';
 import { DownloadLink } from './download-link';
 import { AnalysesResultsManager } from './analyses-results-manager';
 
+const autoDownloadMaxSize = 300 * 1024;
+const autoDownloadMaxCount = 100;
+
 export class RemoteQueriesManager {
   private readonly remoteQueriesMonitor: RemoteQueriesMonitor;
   private readonly analysesResultsManager: AnalysesResultsManager;
+  private readonly interfaceManager: RemoteQueriesInterfaceManager;
 
   constructor(
     private readonly ctx: ExtensionContext,
     private readonly logger: Logger,
     private readonly cliServer: CodeQLCliServer
   ) {
     this.analysesResultsManager = new AnalysesResultsManager(ctx, logger);
+    this.interfaceManager = new RemoteQueriesInterfaceManager(this.ctx, this.logger, this.analysesResultsManager);
```

Comment:
nit: looks a little odd to use `this` on this line, but not on the line above. But, this is purely aesthetics.

```suggestion
    this.interfaceManager = new RemoteQueriesInterfaceManager(ctx, logger, this.analysesResultsManager);
```